apps/accelerate/open_alpha_tensor/open_alpha_tensor/core/actors/stage.py

```diff
@@ -91,9 +99,8 @@ def extract_children_states_from_actions(
             )
             for i in range(k)
         ],
-        old_idx_to_new_idx,
         repetition_map,
-        not_duplicate_indexes,
+        duplicate_indexes,
     )
```

Comment:
please keep the old code modification (the code in red should replace the code in green)